packages/main/src/Slider.js

```diff
@@ -0,0 +1,457 @@
+		 * <br><br>
+		 *
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @public
+		 */
+		 tickmarks: {
+			type: Boolean,
+		},
+		/**
+		 * Defines whether the <code>ui5-slider</code> is in disabled state.
+		 * <br><br>
+		 *
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @public
+		 */
+		disabled: {
+			type: Boolean,
+		},
+	},
+	slots: /** @lends sap.ui.webcomponents.main.Slider.prototype */ {
+		/**
+		 * Defines the text of the <code>ui5-range-slider</code>.
+		 * <br><b>Note:</b> Although this slot accepts HTML Elements, it is strongly recommended that you only use text in order to preserve the intended design.
+		 *
+		 * @type {Node[]}
+		 * @slot
+		 * @public
+		 */
+		"default": {
+			type: Node,
+		},
+	},
+	events: /** @lends sap.ui.webcomponents.main.Slider.prototype */ {
+		/**
+		 * Fired when the value changes and the user has finished interacting with the slider.
+		 *
+		 * @event
+		 * @public
+		*/
+		change: {},
+		/**
+		 * Fired when the value changes due to user interaction.
+		 *
+		 * @event
+		 * @public
+		*/
+		input: {},
+	},
+};
+
+/**
+ * @class
+ *
+ * <h3 class="comment-api-title">Overview</h3>
+ *
+ * <code>import "@ui5/webcomponents/dist/Label";</code>
+ *
+ * @constructor
+ * @author SAP SE
+ * @alias sap.ui.webcomponents.main.Slider
+ * @extends sap.ui.webcomponents.base.UI5Element
+ * @tagname ui5-range-slider
+ * @public
+ */
+class Slider extends UI5Element {
+	static get metadata() {
+		return metadata;
+	}
+
+	static get render() {
+		return litRender;
+	}
+
+	static get template() {
+		return SliderTemplate;
+	}
+
+	static get styles() {
+		return SliderStyles;
+	}
+
+	static async onDefine() {
+		await Promise.all([
+			fetchI18nBundle("@ui5/webcomponents"),
+		]);
+	}
+
+	constructor() {
+		super();
+		this.i18nBundle = getI18nBundle("@ui5/webcomponents");
+	}
+
+	_handleChange() {
+
+	}
+
+	_handleInput() {
+
+	}
+
+	// TODO Include touch/pointer events, keyboard handling
+	/**
+	 * Called when the user starts interacting with the slider
+	 */
+	_handleDown(event) {
+		const _this = this;
+
+		if (this.disabled) {
+			return;
+		}
+
+		const oldEndValue = _this.endValue;
+		const oldStartValue = _this.value;
+		const clientX = event.clientX != null ? event.clientX : event.targetTouches[0].clientX;
+
+		_this.boundingDOMRect = _this.getBoundingClientRect();
+		const newValue = _this._calculateValueFromInteraction(event);
+
+
+		// Do not update value if press is in range (only for range sliders (when valueEnd property is set)
+		if (oldEndValue && newValue >= oldStartValue && newValue <= oldEndValue) {
+			return;
+		}
+
+		// In case of Range Slider assign the handle clossest to the press point, otherwise - the single handle's DOM
+		_this.handle = _this._getClosestHandle(clientX, newValue);
+
+		const moveHandler = moveEvent => {
+			_this._handleMove(moveEvent);
+		};
+		const upHandler = () => {
+			_this._handleUp();
+			_this.removeEventListener("mouseup", upHandler);
```

Comment:
The "mouseup" is added to the body, but removed on the component

Should not we have
document.body.removeEventListener("mouseup", upHandler); 
as you do for the "mousemove"